astroquery/sha/core.py

```diff
@@ -145,6 +145,9 @@ def parse_line(line, cs=cs):
     # To table
     # transpose data for appropriate table instance handling
     t = Table(list(zip(*data)), names=col_names, dtype=dtypes)
+    if len(data) < 0:
+        warnings.warn(NoResultsWarning("No matching rows were found"))
```

Comment:
Hi @HarrietAkot, nice job on your first PR. Can you clarify this to me? I see your using `warnings` and `NoResultsWarnings` but I don't see where they're defined or imported. See this sample file https://github.com/astropy/astroquery/blob/e49a63b9947e448ae71ee08c5261adca283d0ae7/astroquery/mast/cutouts.py. You can search for `warnings` and `NoResultsWarning` and you'll see how they're imported.

Reply:
Thank you @daisycodes for making code review!